### PR TITLE
WIP DONT MERGE Feat/table component

### PR DESCRIPTION
**Pull request by gustav-pettersson-nordnet**

### CODE REVIEW

File: src/Molecules/FlexTable/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      type: ACTION_SET_INITIAL_SORTING,
+      sortOrder,
+      controlledSort,
+    });
+  }, []);
+
+  useEffect(() => {
+    // If the sortOrder changes from the outside, update internal the column sort state
+    if (controlledSort) {
+      // @ts-ignore
```
Comment (crewneck) [inline src/Molecules/FlexTable/Header/Header.tsx:63]: What kind of error do you get here?

Comment (crewneck) [reply src/Molecules/FlexTable/Header/Header.tsx:63]: Maybe add a todo?

Comment (smithsamyan) [reply src/Molecules/FlexTable/Header/Header.tsx:63]: Good idea. I think it's sortOrderProp where's it can be undefined or something. Will look into it :)

**Comment by gustav-pettersson-nordnet** [comment]
Closing WIP

**Pull request merged**